src/prefect/events/clients.py

```diff
@@ -331,20 +330,18 @@ async def _reconnect(self) -> None:
             message: Dict[str, Any] = orjson.loads(await self._websocket.recv())
             logger.debug("  auth result %s", message)
             assert message["type"] == "auth_success", message.get("reason", "")
-        except (AssertionError, ConnectionClosedError) as e:
-            if isinstance(e, AssertionError) or e.code == WS_1008_POLICY_VIOLATION:
-                reason = None
-                if isinstance(e, AssertionError):
-                    reason = e.args[0]
-                elif isinstance(e, ConnectionClosedError):  # pragma: no branch
-                    reason = e.reason
-
-                raise Exception(
-                    "Unable to authenticate to the event stream. Please ensure the "
-                    "provided api_key you are using is valid for this environment. "
-                    f"Reason: {reason}"
-                ) from e
-            raise  # pragma: no cover
+        except AssertionError as e:
```

Comment:
😍

Reply:
Why I did this so goofily the first time is beyond me :D